data/json/ranged_balance.json

```diff
@@ -0,0 +1,9 @@
+[
+    {
+        "type": "EXTERNAL_OPTION",
+        "name": "DISPERSION_PER_GUN_DAMAGE",
+        "info": "Value that adds to weapon dispersion per weapon damage level.",
```

Comment:
At first, after reading this description, I thought that weapon dispersion is affected by its state - say, the more it damaged, the higher the dispersion. I'd change "damage level" to "damage value" for clarification.